addons/cni-cilium/cilium.yaml

```diff
@@ -176,13 +200,59 @@ data:
   cluster-pool-ipv4-cidr: "{{.Config.ClusterNetwork.PodSubnet }}"
   cluster-pool-ipv4-mask-size: "24"
```

Comment:
TODO: This is maybe not correct.

Reply:
`{{.Config.ClusterNetwork.PodSubnet }}` looks good to me here.
Going forward it may be helpful to also make mask-size configurable as well, but if it was not demanded up til now, the default be fine.